### PR TITLE
White ordeals tweak

### PR DESCRIPTION
**Pull request by EgorDinamit**
## About The Pull Request

Essentially buffs certain aspects of all fixers.
Additionally *probably* fixes Claw's white slash.

## Why It's Good For The Game

Balance + Fix

### CODE REVIEW

File: code/modules/mob/living/simple_animal/hostile/ordeal/white.dm [hunk 1]
```diff
@@ -349,10 +350,10 @@
 	maxHealth = 3000
 	health = 3000
 	melee_damage_type = RED_DAMAGE
-	rapid_melee = 2
+	rapid_melee = 6
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/hostile/ordeal/white.dm:353]: ...why triple it's melee damage?

File: code/modules/mob/living/simple_animal/hostile/ordeal/white.dm [hunk 2]
```diff
@@ -486,8 +487,8 @@
 	maxHealth = 4000
 	health = 4000
 	melee_damage_type = PALE_DAMAGE
-	melee_damage_lower = 35
-	melee_damage_upper = 45
+	melee_damage_lower = 50
+	melee_damage_upper = 60
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/hostile/ordeal/white.dm:491]: Given all the other damage he has, all of which is AoE, i don't think he needs an even stronger melee. He already hits for 70-90% max hp damage, and given we have a whopping 5 armors with Pale 8+ (not including realized, we get +7 with Realized w/o the update), this dude hits like a truck regardless of level. Honestly if you actually want to improve Pale Fixer's fight experience, give him more start-up for his multi-slash. It hits about as hard as the beam but has a 0.4 second startup vs the beam's 1.5 second start up.

Tldr; Raw buffing Pale Fixer's melee damage isn't going to make him or more challenging fight, it's just gonna push people into using guns even more because everything obliterates them in melee regardless of armor.

File: code/modules/mob/living/simple_animal/hostile/ordeal/white.dm [hunk 3]
```diff
@@ -403,12 +404,12 @@
...
 		return
-	if(prob(50) && (get_dist(src, target) < multislash_range) && (multislash_cooldown < world.time))
-		MultiSlash(target)
-		return
 	if(prob(80) && (beam_cooldown < world.time))
 		LaserBeam(target)
 		return
+	if(prob(50) && (get_dist(src, target) < multislash_range) && (multislash_cooldown < world.time))
+		MultiSlash(target)
+		return
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/hostile/ordeal/white.dm:412]: I like the change of making the beam more common than the slash by raising it's priority, having fought it I found the slash was very hard to punish AT ALL due to the rapid flick between being in front and behind you.

File: code/modules/mob/living/simple_animal/hostile/ordeal/white.dm [hunk 4]
```diff
@@ -364,13 +365,13 @@
...
-	var/multislash_cooldown_time = 5 SECONDS
+	var/multislash_cooldown_time = 4 SECONDS
 	var/multislash_damage = 75
 	var/multislash_range = 6
 	var/beam_cooldown
-	var/beam_cooldown_time = 15 SECONDS
+	var/beam_cooldown_time = 10 SECONDS
 	/// Red damage dealt on direct hit by the beam
-	var/beam_damage = 300
+	var/beam_damage = 400
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/hostile/ordeal/white.dm:374]: I don't think the beam needs both a 30% increase in damage AND a massive reduction in his post-fire stun.

File: code/modules/mob/living/simple_animal/hostile/ordeal/white.dm (hunk 1 above)
Comment (LanceSmites328) [reply code/modules/mob/living/simple_animal/hostile/ordeal/white.dm:353]: With all the extra ability spam this gives him, i really don't think he needs a TRIPLED attack speed on top of that. That puts him at 210-270 red damage every 2 seconds. So you're essentially being forced to face tank Goodbye every 4 seconds if you want to melee this guy. That's ludicrous.

File: code/modules/mob/living/simple_animal/hostile/ordeal/white.dm [hunk 5]
```diff
@@ -11,8 +11,9 @@
 	health = 3000
 	melee_damage_type = BLACK_DAMAGE
 	rapid_melee = 2
-	melee_damage_lower = 30
-	melee_damage_upper = 40
+	melee_damage_lower = 50
```
Comment (LanceSmites328) [inline code/modules/mob/living/simple_animal/hostile/ordeal/white.dm:14]: i don't know if making this guy a more brutal straight up fight is... great for him overall? A lot of his danger comes from infinite breaching abnos, this change makes him a massive melee bruiser on top of that.

Comment (LanceSmites328) [reply code/modules/mob/living/simple_animal/hostile/ordeal/white.dm:14]: I'll clarify here Im down with the hammer changes because the hammer is a fairly reactable part of the fight that lets you hit him in response to him using a powerful move, something that's being heavily reduced for Red Fixer in this same PR.

File: code/modules/mob/living/simple_animal/hostile/ordeal/white.dm (hunk 2 above)
Comment (EgorDinamit) [reply code/modules/mob/living/simple_animal/hostile/ordeal/white.dm:491]: Will consider.

File: code/modules/mob/living/simple_animal/hostile/ordeal/white.dm (hunk 4 above)
Comment (EgorDinamit) [reply code/modules/mob/living/simple_animal/hostile/ordeal/white.dm:374]: It has a terribly long pre-fire duration.

File: code/modules/mob/living/simple_animal/hostile/ordeal/white.dm (hunk 5 above)
Comment (EgorDinamit) [reply code/modules/mob/living/simple_animal/hostile/ordeal/white.dm:14]: He does not breach as much as he used long ago, and basing the entire ordeal around it is silly, unless we buff the breach ability again.

**Comment by EgorDinamit** [comment]
Reduced melee damage for fixers.

File: code/modules/mob/living/simple_animal/hostile/ordeal/white.dm (hunk 4 above)
Comment (LanceSmites328) [reply code/modules/mob/living/simple_animal/hostile/ordeal/white.dm:374]: > It has a terribly long pre-fire delay.

Yeah but you're buffing the effectiveness of it while obliterating it's punishment time. The wind-up can't reasonably be used to punish it... unless you're using a gun. Wiping out the cooldown after the beam (2 seconds to 0.5) means this guy has practically no time where he isn't either hitting you or blinking past you.

**Review by LanceSmites328 — APPROVED** [review]
Please keep an eye on this to see if this needs to be toned down later.
